guides/pages/how-to-test-webhooks-on-localhost.mdx

```diff
@@ -0,0 +1,280 @@
+---
+meta:
+  title: "How to test webhooks on localhost"
+  description:
+    "Learn how to test Liveblocks webhooks on your local system using
+    localtunnel or ngrok."
+---
+
+Testing webhooks on your local system can be difficult, but there are ways to
+make it possible using tools such as
+[`localtunnel`](https://www.npmjs.com/package/localtunnel) and
+[`ngrok`](https://www.npmjs.com/package/ngrok).
+
+## Create an endpoint in your project
+
+The first step in testing webhooks is making sure you have an API endpoint set
+up in your project. This is the endpoint that’ll receive the webhook event from
+Liveblocks.
+
+In order to use webhooks, we’ll need to retrieve the `headers` and `body` from
+the request. Here’s the basic endpoint we’ll be starting from:
+
+```ts
+export async function POST(request: Request) {
+  const body = await request.json();
+  const headers = request.headers;
+
+  // Handle webhooks
+  // ...
+
+  return new Response(null, { status: 200 });
+}
+```
+
+Create this endpoint in your project, and make it available on `localhost` at
+the following URL:
+
+```
+/api/liveblocks-database-sync
```

Comment:
Since the guide is generic I think we should not mention a single particular use-case.

```suggestion
/api/liveblocks-webhook
```